fmriprep/workflows/bold/outputs.py

```diff
@@ -582,8 +579,7 @@ def init_ds_bold_native_wf(
         name='inputnode',
     )
 
-    raw_sources = pe.Node(niu.Function(function=_bids_relative), name='raw_sources')
-    raw_sources.inputs.bids_root = bids_root
+    raw_sources = pe.Node(BIDSURI(), name='raw_sources')
```

Comment:
do `dataset_links` and `out_dir` need to be defined?

Reply:
🤦 wow... just wow... I only provided inputs to one of them, but there are five! That's just silly.